docs/source/testing/mocking.md

```diff
@@ -7,26 +7,30 @@ Mocking enables Apollo Server to return simulated data for GraphQL operations ba
 
 Mocking enables frontend developers to build out and test UI components and features without needing to wait for a full backend implementation. Mocking is also valuable when using a UI tool like [Storybook](https://storybook.js.org/), because you don't need to start a real GraphQL server.
```

Comment:
This file is another case of "maintaining docs for somebody else's library" but eh, it's the first version we are removing the integration, seems reasonable enough. That said I think we should more explicitly link to graphql-tools docs for more information as this page is not exhaustive (and should not be).

Reply:
Oooh I like that! I added a callout to the beginning of this article:

>  📣 **New in Apollo Server 4:** Apollo Server 4 removes both the [`mocks` and `mockEntireSchema`](../migration/#mocks-and-mockentireschema) constructor options. This article has been updated to use the `@graphql-tools` package to mock data for Apollo Server. For the most up-to-date information on `@graphql-tools`, we recommend referencing [their documentation](https://www.graphql-tools.com/docs/mocking).